protos/memcache/errors.go

```diff
@@ -0,0 +1,41 @@
+package memcache
+
+import (
+	"errors"
+)
+
+// error definitions
+
+// TODO: move some to libbeat?
```

Comment:
I would more see it generalised in packetbeat as libbeat should not know about ports etc, that is specific to packetbeat.